tests/testCoaddInputs.py

```diff
@@ -207,6 +207,7 @@ def getCoaddPath(self, version):
         return os.path.join(self.dataDir,
                             "testCoaddInputs_coadd_with_version_{}_data.fits".format(version))
 
+    @unittest.skip("Depends on BaseRecord get support, which will be changed in DM-8716.")
```

Comment:
These skips were [requested](https://lsstc.slack.com/archives/dm-pybind11-porting/p1484959013000153) by @TallJimbo, in response to my asking why these methods used getL, setD, etc.